### DISCUSSION TITLE
插入list后实际是插入成功，但是ExecuteAffrows返回是0

### DISCUSSION THREAD

**Discussion by cejpo001** [dotnetcore/FreeSql#1254] [Q&A]
List<T> datalist=.......
_freeSql.Insert(datalist).ExecuteAffrows()执行后，实际上数据 已经成功插入数据库，但语句执行后返回是0，那么我怎么判断是否插入成功？如果用ExecuteInserted的话返回插入的值来判断，在一定程度增加了开销，毕竟我只想知道有没有插入成功而以

**Comment by 2881099** [comment]
跟驱动有关，看 ado.net 用的哪个，其实不报错就会成功。

**Comment by cejpo001** [comment]
数据其实是插入成功成，但是函数返回结果是0，所以在程序里判断不了到底有没有执行成功。
FreeSql 3.2.669.0
FreeSql.Provider.SqlServer 3.2.669.0
